packages/contact-center/cc-components/src/components/task/Task/index.tsx

```diff
@@ -28,14 +32,22 @@ const Task: React.FC<TaskProps> = ({
   queue,
   acceptTask,
   declineTask,
-  isBrowser,
+  interactionId,
+  taskSelected,
+  acceptText,
+  disableAccept = false,
+  declineText,
 }) => {
   const capitalizeFirstWord = (str: string) => {
     return str.replace(/^\s*(\w)/, (match, firstLetter) => firstLetter.toUpperCase());
   };
 
   return (
-    <ListItemBase className={`task-list-item ${selected ? 'task-list-item--selected' : ''}`}>
+    <ListItemBase
+      className={`task-list-item ${selected ? 'task-list-item--selected' : ''}`}
+      onPress={taskSelected}
```

Comment:
Can we put a check for taskSelected as it is optional. When I click on this through an incoming task it should not break anything
```suggestion
      onPress={taskSelect? taskSelected:undefined}
```

Reply:
addressed